db/migrate/20230404205721_create_post.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: {to_table: :users}
+      t.string :title
```

Comment:
- [ ] Good job here creating the reference foreign key as `author_id` this is the correct way to do it.

Reply:
hello, nice to meet you, sorry but, I think then there is an error in the documentation that we have seen because I attached the image of the documentation that I observed from microverse 
<img width="500" alt="Captura de pantalla 2023-04-04 a la(s) 16 48 52" src="https://user-images.githubusercontent.com/99061965/229940678-1a43ed11-e2de-4ab7-a09a-6b7cd8fe42af.png">